src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyDiscoveryManager.cs

```diff
@@ -99,17 +99,7 @@ public void DiscoverTests(DiscoveryCriteria discoveryCriteria, ITestDiscoveryEve
         // marked as NotDiscovered.
         _dataAggregator.MarkSourcesWithStatus(discoveryCriteria.Sources, DiscoveryStatus.NotDiscovered);
 
-        if (nonRunnableWorkloads.Count > 0)
-        {
-            // We found some sources that don't associate to any runtime provider and so they cannot run.
-            // Mark the sources as skipped.
-
-            _dataAggregator.MarkSourcesWithStatus(nonRunnableWorkloads.SelectMany(w => w.Work.Sources), DiscoveryStatus.SkippedDiscovery);
-            // TODO: in strict mode keep them as non-discovered, and mark the run as aborted.
-            // _dataAggregator.MarkAsAborted();
```

Comment:
Is this really no longer relevant?

Reply:
Tagging @nohwnd for this one.

Reply:
It is relevant, this was a fix that happened since this change was made. So this code should happening somewhere.